src/sentry/grouping/ingest/seer.py

```diff
@@ -187,19 +206,19 @@ def get_seer_similar_issues(
     should go in (if any), or None if no neighbor was near enough.
     """
     event_hash = event.get_primary_hash()
-    stacktrace_string = get_stacktrace_string(get_grouping_info_from_variants(variants))
     exception_type = get_path(event.data, "exception", "values", -1, "type")
 
     request_data: SimilarIssuesEmbeddingsRequest = {
         "event_id": event.event_id,
         "hash": event_hash,
         "project_id": event.project.id,
-        "stacktrace": stacktrace_string,
+        "stacktrace": event.data.get("stacktrace_string", ""),
```

Comment:
Do we need to use `.get` here? If we make it to this point, we know we'll have stored the string on the event.

Reply:
we don't need it, but considering that it's in ingest I thought we'd play it safe just in case someone calls this function again later and forgets this caveat

Reply:
That fair. Actually, here's an idea: Assuming my tallying-of-frames business works, I'm realizing I'm actually still going to need the variants to be passed into this function. So what if we were to do it like this?

```suggestion
        "stacktrace": event.data.get(
            "stacktrace_string", get_stacktrace_string(get_grouping_info_from_variants(variants))
        ),
```

Then you also wouldn't need to add a call to `should_call_seer_for_grouping` in `test_sends_expected_data_to_seer`.